AutomatedTesting/Gem/PythonTests/AWS/Windows/core/test_aws_resource_interaction.py

```diff
@@ -3,29 +3,43 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+Prerequisites for running AWS automation tests on your local Windows machine:
+1) Go to the AWS IAM console and create an IAM role called o3de-automation-tests which adds your own AWS account as
```

Comment:
nit: Same comment as before.

It does seem like this is a common set of setup instructions, could we avoid duplicating these?